src/Dfc.CourseDirectory.WebV2/Features/NewApprenticeshipProvider/ProviderDetailInfoPanel.cshtml

```diff
@@ -0,0 +1,53 @@
+﻿@model Dfc.CourseDirectory.WebV2.Features.NewApprenticeshipProvider.ProviderDetailInfoPanel.ViewModel
+
+    <h2 class="govuk-heading-m" style="margin-bottom:0px">Contact details</h2>
+<aside class="pttcd-apprenticeship-qa-info-panel pttcd-apprenticeship-qa-info-panel--first" role="complementary">
```

Comment:
Don't re-use pttcd-apprenticeship-qa-info-panel styles here - it's a different component